bin/concat_tsv.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
Can this be done by an nf-core module? Maybe: https://nf-co.re/modules/csvtk_concat/? To reduce the number of custom code and maintenance load :P

Reply:
Actually, this was replaced with the custom script to reduce the memory usage for large datasets by processing it chunk-wise (`csvtk` loads everything into memory). good point, maybe this can be mentioned somewhere

Reply:
I see! Then lazy reading with `chunksize` is a great choice 😎